tests/functional/lib/python/diffr.py

```diff
@@ -40,10 +40,6 @@
 import json
 import sys
 
-import colorama
-
-colorama.init(strip=True)
-
```

Comment:
Unused?

Reply:
I think colour was meant for interactive use but for some reason those lines were commented out. I don't think anyone uses this interactively so whatever.